docker/rootfs/usr/local/go2rtc/sbin/go2rtc.yaml

```diff
@@ -0,0 +1,6 @@
+rtsp:
+  listen: ":554"
```

Comment:
The reason most apps use 8554 is because ports below 1024 are reserved for the root user. I think it makes sense to stick with 8554 since its typical for containers to bind to ports that don't require root.

Reply:
Ah I see, that makes sense 👍

Reply:
Well, the Frigate container runs as root.